pkg/forwarder/telemetry.go

```diff
@@ -22,6 +22,7 @@ var (
 	v1IntakeEndpoint       = transaction.Endpoint{Route: "/intake/", Name: "intake"}
 	v1SketchSeriesEndpoint = transaction.Endpoint{Route: "/api/v1/sketches", Name: "sketches_v1"} // nolint unused for now
 	v1ValidateEndpoint     = transaction.Endpoint{Route: "/api/v1/validate", Name: "validate_v1"}
+	v1MetadataEndpoint     = transaction.Endpoint{Route: "/api/v1/metadata", Name: "metadata_v1"}
```

Comment:
I'm a little unclear why there's a "new" v1 endpoint when we already have a v2 metadata endpoint?

Reply:
The v2 metadata endpoint doesn't actually exist (the "v2 metadata code" the Agent was added a long time ago, but should be cleaned up as the endpoint never actually existed)

Reply:
OK, I'll make a PR to remove that.